Resources/installer/sql/demo_data.sql

```diff
@@ -1399,6 +1398,7 @@ INSERT INTO `ezcontentobject_attribute` VALUES (0,332,126,0,NULL,'<?xml version=
 INSERT INTO `ezcontentobject_attribute` VALUES (0,333,126,0,NULL,'','ezcountry',787,'eng-GB',2,0,'',1);
 INSERT INTO `ezcontentobject_attribute` VALUES (0,334,126,0,NULL,'','ezkeyword',788,'eng-GB',2,0,'',1);
 INSERT INTO `ezcontentobject_attribute` VALUES (0,335,126,0,0,'','ezgmaplocation',789,'eng-GB',2,0,'',1);
+INSERT INTO `ezcontentobject_attribute` VALUES (0,336,126,NULL,NULL,'<?xml version=\"1.0\" encoding=\"utf-8\"?>\n<section xmlns:image=\"http://ez.no/namespaces/ezpublish3/image/\" xmlns:xhtml=\"http://ez.no/namespaces/ezpublish3/xhtml/\" xmlns:custom=\"http://ez.no/namespaces/ezpublish3/custom/\"/>\n','ezxmltext',790,'eng-GB',2,0,'',1);
```

Comment:
@pspanja but the content should be different as well right? (not `//ez.no/namespaces/ezpublish3/custom/` as we use docbook in `ezrichtext`)